evalml/tests/automl_tests/test_automl.py

```diff
@@ -4040,7 +4040,7 @@ def test_automl_baseline_pipeline_predictions_and_scores_time_series(problem_typ
         expected_predictions = pd.Series(expected_predictions, name="target_delay_1")
 
     preds = baseline.predict(X_validation, None, X_train, y_train)
-    pd.testing.assert_series_equal(expected_predictions, preds)
+    pd.testing.assert_series_equal(expected_predictions, preds, check_dtype=False)
```

Comment:
is the dtype no longer matching in the datasets?

Reply:
Changed the casting above to include `time_series_regression`, removed the dtype check, thanks!